recipes-devtools/cargo/cargo_0.13.0.bb

```diff
@@ -0,0 +1,125 @@
+require cargo-snapshot.inc
+require cargo.inc
+
+SRC_URI += " \
+	git://github.com/rust-lang/cargo.git;protocol=https;name=cargo \
+	file://0001-disable-cargo-snapshot-fetch.patch \
+	git://github.com/rust-lang/rust-installer.git;protocol=https;name=rust-installer;destsuffix=${S}/src/rust-installer \
+	crate://crates.io/advapi32-sys/0.2.0 \
+	crate://crates.io/aho-corasick/0.5.2 \
+	crate://crates.io/bitflags/0.1.1 \
+	crate://crates.io/bitflags/0.7.0 \
+	crate://crates.io/bufstream/0.1.2 \
+    crate://crates.io/cfg-if/0.1.0 \
+	crate://crates.io/cmake/0.1.17 \
+	crate://crates.io/crossbeam/0.2.9 \
+	crate://crates.io/docopt/0.6.82 \
+    crate://crates.io/env_logger/0.3.4 \
+	crate://crates.io/filetime/0.1.10 \
+	crate://crates.io/flate2/0.2.14 \
+	crate://crates.io/fs2/0.2.5 \
+	crate://crates.io/gcc/0.3.32 \
+	crate://crates.io/gdi32-sys/0.2.0 \
+	crate://crates.io/glob/0.2.11 \
+	crate://crates.io/hamcrest/0.1.0 \
+	crate://crates.io/idna/0.1.0 \
+	crate://crates.io/kernel32-sys/0.2.2 \
+    crate://crates.io/lazy_static/0.2.1 \
```

Comment:
Actually, lots of spacing issues... check the rest of this list, because there are more.